features/project_test.feature

```diff
@@ -0,0 +1,286 @@
+@api @test
+Feature: Test
+
+  Users should be able to retrieve the test details by its id.
+
+  The details of a test contains:
+  - name
+  - category
+  - tags
+  - key
+  - project (object)
+  - author/contributors
+  - number of results
+  - first run date
+  - last run date
+  - latest project version
+  - passing (last result)
+  - active/inactive
+
+
+
+  Scenario: An organization member should be able to get the test details without project data in his organization
+    Given private organization Rebel Alliance exists
+    And user hsolo who is a member of Rebel Alliance exists
+    And user lskywalker who is a member of Rebel Alliance exists
+    And user wantilles who is a member of Rebel Alliance exists
+    And project X-Wing exists within organization Rebel Alliance
+    And test "Ion engine should provide thrust" was created by lskywalker with key k1 for version 1.0.0 of project X-Wing
+    And test "Ion engine should provide thrust" has category C1 and tags tag1, tag2 and tickets ticket1, ticket2 for version 1.0.0
+    And test "Blasters should fire" was first run by lskywalker for version 1.0.0 of project X-Wing
+    And test "Blasters should fire" has category C2 for version 1.0.0
+    When hsolo sends a GET request to /api/tests/{@idOf: Ion engine should provide thrust}?organizationId={@idOf: Rebel Alliance}
```

Comment:
Why is there an `organizationId` query param in this GET request? Same question for the other tests (sometimes with `projectId`)? It should not be necessary to add these parameters for a GET on a single resource.